### PR TITLE
Added reward that gets calculated in stroll to the stroll screen using the constructor of the reward screen. Modifications to the Screenlogic had to be made.

### PR DESCRIPTION
**Pull request by sirpolomotro**
Added reward that gets calculated in stroll to the stroll screen using the constructor of the reward screen. Modifications to the Screenlogic had to be made.

### CODE REVIEW

File: core/src/cg/group4/view/screen/EventScreen.java [hunk 1]
```diff
@@ -15,11 +15,14 @@
      */
     protected Label cTaskToPerform;
 
+    public EventScreen() {
+        cTaskToPerform = new Label("TASK:", cGameSkin.get("default_labelStyle", Label.LabelStyle.class));
```
Comment (Surskit) [inline core/src/cg/group4/view/screen/EventScreen.java]: No Javadoc

File: core/src/cg/group4/view/screen/RewardScreen.java [hunk 2]
```diff
@@ -13,12 +13,18 @@
  */
 public final class RewardScreen extends ScreenLogic {
 
+    protected int cReward;
+
```
Comment (Surskit) [inline core/src/cg/group4/view/screen/RewardScreen.java]: Missing javadoc

**Comment by sirpolomotro** [comment]
Added the missing javadoc, good call.

**Comment by Jurgenvs** [comment]
Compiled and played, looked fine. The code also looks good, no extra comments to make to those that have already been made.

**Pull request merged**